dataInput/input.py

```diff
@@ -1,8 +1,8 @@
-def print_input():
-    print("Hello from input")
 class dataReceiver:
     def __init__(self, argument1, argument2):
         return
+    def __init__(self):
+        return
 
     def print_input():
```

Comment:
Add self to arguments to fix error:
"function takes 0 positional arguments but 1 was given"